api/src/db/models/competition_models.py

```diff
@@ -324,6 +324,27 @@ def download_path(self) -> str:
         return pre_sign_file_location(self.file_location)
 
 
+class ShortLivedInternalToken(ApiSchemaTable, TimestampMixin):
+    __tablename__ = "short_lived_internal_token"
+
+    short_lived_internal_token_id: Mapped[uuid.UUID] = mapped_column(
+        UUID, primary_key=True, default=uuid.uuid4
+    )
+
+    application_id: Mapped[uuid.UUID] = mapped_column(
+        UUID, ForeignKey(Application.application_id), nullable=False, index=True
+    )
+    application: Mapped[Application] = relationship(Application)
+
+    form_id: Mapped[uuid.UUID] = mapped_column(
+        UUID, ForeignKey(Form.form_id), nullable=False, index=True
+    )
+    form: Mapped[Form] = relationship(Form)
```

Comment:
We don't need any reference to these tables - while the token will be used for fetching that info, no need to complicate it (I think I was planning to just reuse a token for all forms in an app)